lib/index.js

```diff
@@ -17,7 +17,7 @@ module.exports = strapi => {
       applicationId: '',
       apiKey: '',
       debug: false,
-      prefix: strapi.config.environment,
```

Comment:
I don't think `false` should be the default value here. I think it is a best practice to use environments in the index names.
So no need to change it here and in the [README.md](https://github.com/MattieBelt/strapi-hook-algolia/blob/86d0f1e4d610c2959e4a541526fac176981e1ee7/README.md).